### PR TITLE
Tweak layout

### PR DESCRIPTION
**Pull request by boyleconnor**
Made some minor changes to the layout and consistently added a hyphen to "de-identifier".
New Layout

### CODE REVIEW

**Comment by tschaffter** [comment]
Thanks for the update!

Here are other improvements to consider:

- Use a human-friendly label for the type of annotation, e.g. "Date" instead of "text_date", "Person name" instead of "text_person_name" and "Physical address" instead of "text_physical_address"
- The option "Method" is not very descriptive. Maybe "Masking method" or "Obfuscation method"?
- I think that we should make the text areas slightly larger. They should be at least 100 or 120 characters wide.
- Add a message that says:

> This application is for demonstration only. Only submit PHI information to a server that your organization has approved.

**Comment by tschaffter** [comment]
@cascadianblue I cloned your branch and did `docker-compose up --build` and got this error:

image

**Comment by tschaffter** [comment]
From the screenshot, it looks like you didn't replace the annotation type by something more user-friendly. Is this an oversight?

**Review by tschaffter — CHANGES_REQUESTED** [review]
See comment in discussion

**Comment by cascadianblue** [comment]
> @cascadianblue I cloned your branch and did `docker-compose up --build` and got this error:

Could you please tell me what error shows up in your browser console? I can't reproduce

**Comment by tschaffter** [comment]
It's the CORS issue. What I did.

1. Fresh clone of the repo
2. cp .env.example .env
3. docker-compose up --build

```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://localhost:8080/api/v1/tool. (Reason: CORS request did not succeed).

Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://localhost:8080/api/v1/tool/dependencies. (Reason: CORS request did not succeed).
```

If you are not sure what I may do wrong, can you please try from a clean dev environment? New console, no env vars set.

**Comment by cascadianblue** [comment]
@tschaffter I just made a fresh clone and tried myself. Still couldn't reproduce the bug; the API calls are going through beautifully for me. I can't think of why your client is making requests to `localhost:8080`. That is the wrong port. It should be making requests to `localhost:80`. Could you tell me what the value for `SERVER_PORT` is in your `.env` (and by extension `.env.example`)?

Or better yet, could you do a search at the project root: `grep -r * -e "8080"`?

I get the following result:

```
docker-compose.yml:      - DATE_ANNOTATOR_PORT=8080
docker-compose.yml:      - PERSON_NAME_ANNOTATOR_PORT=8080
docker-compose.yml:      - PHYSICAL_ADDRESS_ANNOTATOR_PORT=8080
docker-compose.yml:      - SERVER_PORT=8080
```

for me, that last value where `SERVER_PORT=8080` is under the `nginx` service in the docker-compose file

**Comment by cascadianblue** [comment]
I would also be interested in seeing what your `http://localhost/env-config.js` looks like.

For me it is:

```
window._env_ = {
  SERVER_PROTOCOL: "http://",
  SERVER_HOST: "localhost",
  SERVER_PORT: "80",
  SERVER_API_PATH: "/api/v1",
}
```

though I suspect for you it would have the line

`SERVER_PORT: "8080"`

**Comment by tschaffter** [comment]
Thanks @cascadianblue ! This is enough evidence for me that the issue may be on my side. I'll take some time to investigate before sharing my findings here.

**Comment by tschaffter** [comment]
I followed the request I sent you (open new terminal, fresh env vars) and now the app is working. :-D

File: client/public/index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>NLP Sandbox PHI Deidentifier</title>
+    <title>NLP Sandbox PHI De-identifier</title>
```
Comment (tschaffter) [inline client/public/index.html:33]: It seems that the preferred title from the poll on Discord is `NLP Sandbox PHI Deidentifier`.

I agree with using "de-identification", "de-identify", etc.

File: client/src/components/InfoDialog.js [hunk 2]
```diff
@@ -131,9 +131,16 @@ export class InfoDialog extends React.Component {
         aria-labelledby="scroll-dialog-title"
         aria-describedby="scroll-dialog-description"
         maxWidth="lg"
+        padding={2}
       >
         <DialogTitle id="scroll-dialog-title">About This Tool</DialogTitle>
-        {content}
+        <DialogContent>
+          <DialogContentText>
+            Note: this application is for demonstration only. Only submit PHI
```
Comment (tschaffter) [inline client/src/components/InfoDialog.js:139]:
```suggestion
            Note: This application is for demonstration only. Only submit PHI
```

**Review by tschaffter — CHANGES_REQUESTED** [review]

**Comment by tschaffter** [comment]
@cascadianblue Can you please comment the following suggestions I made earlier?

- From the screenshot, it looks like you didn't replace the annotation type by something more user-friendly. Is this an oversight?
- I think that we should make the text areas slightly larger. They should be at least 100 or 120 characters wide.

File: client/public/index.html (hunk 1 above)
Comment (cascadianblue) [reply client/public/index.html]: Got it. Changed it back now

**Comment by cascadianblue** [comment]
@tschaffter I added in the human-friendly names for the annotation types. Sorry I forgot to add that in earlier! I also fixed the capitalization and hyphen issues.

As for the textbox size, the way it works currently is with each horizontal half using 4 columns each of the 12-column grid (they bump up to 6 each on smaller screens though). I could maybe change it to each one using 6 columns no matter what, but setting some padding dynamically. I'll try to play around with it.

**Comment by cascadianblue** [comment]
> I followed the request I sent you (open new terminal, fresh env vars) and now the app is working. :-D

It's good that you got it working, although I am still quite curious how this bug occurred in the first place! I'm a bit worried that it seemed to materialize out of nowhere.

You referred to it as a "CORS error", which is technically correct. However, to me "CORS Error" sounds like you are trying to get your front end to talk to your API and the browser policy is getting in the way. In your situation, the client was simply making requests to the wrong back end address. CORS getting in the way of this requests happened to actually be the right thing to do!

In this case, it happened to just be a different port of the correct host, but any example of the client accidentally sending requests (of sensitive data) to the wrong host/address puts me on edge. I wonder if it would be a good idea to display the API host/address as part of the `InfoDialog`?

File: client/src/components/InfoDialog.js (hunk 2 above)
Comment (tschaffter) [reply client/src/components/InfoDialog.js]: Implemented

**Comment by tschaffter** [comment]
> I am still quite curious how this bug occurred in the first place!

My best bet is that I used the same terminal to develop the Data Node, which has environment variables that have the same name as this app.

**Pull request merged**